### PR TITLE
Create animal database

### PR DESCRIPTION
**Pull request by clin2on3mun**
in this milestone : 

- create a database called vet_clinic.
- create animals table with columns(id, name, date_of_birth, escape_attempts, neutered, weight_kg).
-Insert data in table which is data.sql file.
-create queries in queries.
Result of queries and there queries:
Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.

sql2

List the name of all animals born between 2016 and 2019.

sql3

List the name of all animals that are neutered and have less than 3 escape attempts.

sql4
List the date of birth of all animals named either "Agumon" or "Pikachu".

Sql5

List name and escape attempts of animals that weigh more than 10.5kg

sql6

Find all animals that are neutered.

sql7

Find all animals not named Gabumon.

sql8

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

sql9

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1,'Agumon', 'feb-3-2020', 0, true, 10.23);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered, weight_kg) VALUES (2, 'Gabumon', 'Nov-15-2018', 2, true, 8);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered, weight_kg) VALUES (3, 'Pikachu', 'Jan-7-2021', 1, false, 15.04);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered, weight_kg) VALUES (3, 'Devimon', 'May-12-2017', 5, true, 11);
+
```
Comment (Qoosim) [inline data.sql:7]: - [ ] There's a spelling error in your data.sql file. You declared `escape_attempt` in your schema file NOT `escape_attempts`. Kindly remove the `s` to fix the error.

### Error
error

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @clin2on3mun, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Fair README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
+    name VARCHAR,
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOL,
+    weight_kg DECIMAL,
+    PRIMARY KEY(id)
 );
```
Comment (abdulhamiid) [inline schema.sql]: - [ ] As requested by the previous reviewer concerning the spelling error in your `animals` table

image

- [ ] According to the project requirements 👀

image

image

- [ ] Kindly copy the SQL you used to rename the column into the schema.sql file 🤓👍

- [ ] To rename a column of a table, we use the [ALTER TABLE](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-alter-table/) statement with RENAME COLUMN clause as follows:

```SQL
ALTER TABLE table_name 
RENAME COLUMN column_name TO new_column_name;
```

This helps to ensure that you have run the queries with success. 🟢🛠🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @clin2on3mun 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Create tables in SQL.
- ✔️ Insert and query data in SQL.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @clin2on3mun 👋 

## Things to appreciate
- [ ] Good job for implementing the things requested by the previous code reviewers. 👍 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**